homeassistant/components/melcloud/strings.json

```diff
@@ -16,7 +16,7 @@
       "unknown": "[%key:common::config_flow::error::unknown%]"
     },
     "abort": {
-      "already_configured": "MELCloud integration already configured for this email. Access token has been refreshed."
+      "already_configured": "[%key:common::config_flow::abort::already_configured_account%]"
```

Comment:
Is it OK to replace by the common strings as their is an explication to the user "token has been refreshed". It could be an issue to not explain that to users ?

Reply:
`"already_configured":` AND `already configured for this email` threw me off here.
`invalid_access_token` could be the alternative. 
On further investigation:
`already_configured` is not used in any of the integration files https://github.com/home-assistant/core/tree/dev/homeassistant/components/melcloud only in https://github.com/home-assistant/core/blob/dev/tests/components/melcloud/test_config_flow.py
Skipping this one in this PR.